app/src/views/OperationalLearning/KeyInsights/index.tsx

```diff
@@ -0,0 +1,122 @@
+import {
+    ArrowDownSmallFillIcon,
+    ArrowUpSmallFillIcon,
+} from '@ifrc-go/icons';
+import {
+    Button,
+    Container,
+} from '@ifrc-go/ui';
+import {
+    useBooleanState,
+    useTranslation,
+} from '@ifrc-go/ui/hooks';
+import {
+    formatDate,
+    resolveToString,
+} from '@ifrc-go/ui/utils';
+import { isDefined } from '@togglecorp/fujs';
+
+import Link from '#components/Link';
+import { type GoApiResponse } from '#utils/restRequest';
+
+import Sources from '../Sources';
+
+import i18n from './i18n.json';
+import styles from './styles.module.css';
+
+type OpsLearningSummaryResponse = GoApiResponse<'/api/v2/ops-learning/summary/'>;
+
+interface Props {
+    opsLearningSummaryResponse: OpsLearningSummaryResponse;
+}
+
+function KeyInsights(props: Props) {
+    const {
+        opsLearningSummaryResponse,
+    } = props;
+
+    const strings = useTranslation(i18n);
+
+    const [
+        isExpanded,
+        {
+            toggle: toggleExpansion,
+        },
+    ] = useBooleanState(false);
+
+    return (
+        <Container
+            className={styles.keyInsights}
+            heading={strings.opsLearningSummariesHeading}
+            withInternalPadding
+            withOverflowInContent
+            footerIcons={(
+                <div className={styles.footerIcons}>
+                    <span>
+                        {resolveToString(strings.keyInsightsDisclaimer, {
+                            numOfExtractsUsed: opsLearningSummaryResponse.used_extracts_count,
+                            totalNumberOfExtracts: opsLearningSummaryResponse.total_extracts_count,
+                            appealsFromDate: formatDate(
+                                opsLearningSummaryResponse.earliest_appeal_date,
+                                'MMM-yyyy',
+                            ),
+                            appealsToDate: formatDate(
+                                opsLearningSummaryResponse.latest_appeal_date,
+                                'MMM-yyyy',
+                            ),
+                        })}
+                    </span>
+                    <Link
+                        className={styles.reportIssue}
+                        href="/"
+                        external
+                    >
+                        {strings.keyInsightsReportIssue}
+                    </Link>
```

Comment:
Do we not have the link yet? Let's at least add a TODO

Reply:
We don't have a link as of now. Will add a TODO.